pom.xml

```diff
@@ -65,9 +65,11 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
     <antlr4.version>4.5</antlr4.version>
+    <jcasc.version>1.13</jcasc.version>
+    <access-modifier-checker.failOnError>false</access-modifier-checker.failOnError>
```

Comment:
:-1: we should just copy the bits we need in-tree. Turning off this checker is not the solution

Reply:
done :+1: